centreon/packages/ui/src/Listing/index.tsx

```diff
@@ -140,6 +143,10 @@ const useStyles = makeStyles<StylesProps>()(
       position: 'relative'
     },
     tableBody: {
+      '.MuiTableRow-root > div:first-of-type': {
+        paddingLeft: theme.spacing(1.5)
+      },
+
       display: 'contents',
       'div:first-of-type': {
```

Comment:
same comment here

Reply:
this selector , to select the first columns of all cells of the table , what could be chnged for a table !